pkg/mapping/processors/assetinventory.go

```diff
@@ -119,7 +118,7 @@ func (p *AssetInventoryProcessor) validateAndEnrich(ctx context.Context, resourc
 	}
 	resource, err := p.getSingleResource(ctx, resourceSearchReq)
 	if err != nil {
-		return nil, fmt.Errorf("failed to get single matched resource: %w", err)
+		return nil, pmaperrors.New(fmt.Sprintf("failed to get single matched resource: %v", err))
 	}
```

Comment:
https://github.com/abcxyz/pmap/blob/main/pkg/mapping/processors/assetinventory.go#L86 should also be user facing error